packages/fe-fpm-writer/test/unit/section.test.ts

```diff
@@ -124,6 +124,88 @@ describe('CustomSection', () => {
             expect(testFS.read(expectedFragmentPath)).toMatchSnapshot();
         });
 
+        test('no handler, no fs, no folder, section name uppercase', () => {
+            const testCustomSection: CustomSection = {
+                ...customSection,
+                folder: undefined
+            };
+
+            const testFS = generateCustomSection(testDir, { ...testCustomSection });
+            const fragmentPath = join(
+                testDir,
+                `webapp`,
+                `ext`,
+                `newCustomSection`,
+                `${testCustomSection.name}.fragment.xml`
+            );
+            expect(testFS.exists(fragmentPath)).toBeTruthy();
+        });
+
+        test('no handler, no fs, no folder, section name lowercase', () => {
+            const testCustomSection: CustomSection = {
+                ...customSection,
+                folder: undefined,
+                name: 'newCustomSection'
+            };
+
+            const testFS = generateCustomSection(testDir, { ...testCustomSection });
+            const fragmentPath = join(
+                testDir,
+                `webapp`,
+                `ext`,
+                `${testCustomSection.name}`,
+                `${testCustomSection.name}.fragment.xml`
+            );
+            expect(testFS.exists(fragmentPath)).toBeTruthy();
+        });
+
+        test('no handler, no fs, no folder, section name uppercase', () => {
+            const testCustomSection: CustomSection = {
+                ...customSection,
+                folder: undefined,
+                name: 'NewCustomSection'
+            };
+
+            const testFS = generateCustomSection(testDir, { ...testCustomSection });
+            const updatedManifest: any = testFS.readJSON(join(testDir, 'webapp/manifest.json'));
```

Comment:
We discourage the use of `any`. The `project-access` module has a type for `Manifest` that you can use here. Please use it.

Reply:
Not so easy, as the Manifest definition is not detailed enough when it comes to routing.targets (target details = unknown). But I give it a try.

Reply:
Was not so easy, as there is no dedicated definition for routing.targets, but I find some workaround. Please check.